lib/aramanim.py

```diff
@@ -0,0 +1,114 @@
+from manim import UP, LEFT, RIGHT, ORIGIN, PI, DEFAULT_STROKE_WIDTH
+from manim import VMobject, VGroup, Line
+from manim import TexTemplate, MathTex
+from manim import always_redraw
+import numpy as np
+
+# np վեկտորների համար ՛մեթոդներ՛
+
+def length(array):
+    array = np.array(array)
+    return np.matmul(array.transpose(), array) **(1/2)
+
+def normalized_vector(array):
+    array = np.array(array)
+    return (1/length(array)) * array
+
+
+def rotate2angle(array, alpha):
+    array = np.array(array)
+    rotation__matrix = [
+        [np.cos(alpha), -np.sin(alpha), 0],
+        [np.sin(alpha), np.cos(alpha), 0],
+        [0, 0, 1],
+    ]
+    return np.matmul(rotation__matrix, array)
+
+def normal(array):
+    array = np.array(array)
+    return normalized_vector(rotate2angle(array, PI/2))
+
+# հայերեն լեզվով LaTeX աշխատացնելու համար
+
+XELATEX_preamble = r"""
+\usepackage{tikz}
+\usepackage{fontspec}
+\setmainfont{DejaVu Serif}
+\usepackage{amsfonts,amssymb,amsthm,mathtools}
+\usepackage{amsmath}
+\usepackage{upgreek}
+\usepackage{mathrsfs}
+"""
+XELATEX = TexTemplate('xelatex', '.pdf', preamble=XELATEX_preamble)
+
+
+ARMTEX = TexTemplate()
+ARMTEX.add_to_preamble(
+    r"""\usepackage{armtex}
+\usepackage{xcolor}
+"""
+)
```

Comment:
Don't use this template here. It will be the default tex template, if the problem will be rendered in armenian.
Remove all references as well.